src/vaadin-combo-box.html

```diff
@@ -90,6 +91,8 @@
       _item-label-path="[[itemLabelPath]]"
       loading="[[loading]]">
     </vaadin-combo-box-dropdown-wrapper>
+
+    <iron-media-query query="(min-width: 600px)" query-matches="{{_desktopMode}}"></iron-media-query>
```

Comment:
Device width is not a really detecting the input method, is it? :)

I don’t like the fact that we need a new dependency for this (iron-media-query). We should detect touch devices, whether the button was clicked with a finger or a touch device.